save-plugins/warn-plugin/src/commonTest/kotlin/org/cqfn/save/plugin/warn/WarnPluginTest.kt

```diff
@@ -265,4 +266,8 @@ class WarnPluginTest {
             )
         }
     }
+
+    companion object {
+        private const val NAME_WARN = "Some warnings were unexpected: [[Warning(message=Variable name should be in lowerCamelCase, line=5, column=8)]]"
```

Comment:
But this is a text for a one particular test, why should it be a class-level constant?